### ISSUE TITLE
HIGH PRIORITY #21 refactor CSS

### ISSUE THREAD

**Issue by mollyfish** [mollyfish/advanced-presentation-AT-MK-wip#21]

**Comment by toalina** [comment]
A few notes as we discussed earlier:

TOP NAV: Good.

HEADER: Good.

SERVICES: 
- Possible consolidate of strengths icons flex feature with Portfolio figures?
- Double check paragraphs (2 and 3) have same flex feature as About section paragraphs.

PORTFOLIO: 
- See Services notes on figures vs strengths icons
- Any code overlap between Portfolio nav and Top Nav?

ABOUT: 
- See Services notes on paragraphs

TEAM: 
- Overlap on flex feature with strengths icons / Portfolio figures / Team figures?

CONTACT: Good.

FOOTER: Good.

**Comment by mollyfish** [comment]
I have worked my way down to the end of the Portfolio section, converting px to em.  I am taking a dinner break now :) and will pick up with the About section later this evening.  I also tightened up the HTML (removing spaces, nothing else!) so it looks very dense, now.  Hopefully I'll be done with the px sweep before you want to start any major work, so we can merge my edits and then so another round of comparisons.  Sound okay?  I will check for slack and email before I dive in again, in case you have a different plan in mind.

**Comment by toalina** [comment]
Ok! This sounds good. I just got home from dinner and plan to work till pretty late tonight on whatever needs to get done. 

Please let me know when you've messed with the px to em in the About section, pull request, and then I'll go in and mess with some of the flex-container classes.

**Comment by mollyfish** [comment]
I am just about done!  Perfect timing, as usual.  :)

I also got enthusiastic and ran the html thru the validator, and the only warning we have is that some of our sections lack headers, which I REALLY don't think is a problem.  :)

Last thing for me to do is add titles to the tables and hide them if I can (if I can't hide them I won't add them) and then I can hand off to you.  I tried to clean stuff up as I saw it, going through on the px hunt, and I think we are in pretty good shape.  There may be a few duplications, but our file is sooooo long I think leaving them in the context of the section they style would be better than pulling one or two little things into another section just to save a couple lines.  For example, we had some .about h3 styles in the Services part of the css file, and I went ahead and moved them back down to About since I really wasn't expecting to find them there.  Thoughts?

I think trying to consolidate nav styles is still worth doing, I haven't even looked at that.

**Comment by toalina** [comment]
Oh good! I'm glad you ran the HTML Validator. I was gonna wait till you added the <table> attributes. 

I agree with the CSS consolidating. On one hand, it is a LOT of lines in there... On the other hand, it's pretty organized and we both know where stuff is, as they are appropriately placed in there. I'll go through and see what can be moved together, but I don't think I'll bother adding a whole new class and potentially messing up a TON of layout with one tiny change... 

I'll definitely look at the different NAVs. Just the TOP NAV and PORTFOLIO sections. 

Anything else you think of and need me to look through. Since you've been awake since God knows when, you deserve some horizontal lay down time. I'll mess with this thing tonight till I get sick of it. We can check in again in the AM. I have till about 11AM tomorrow morning, then I have stuff till pretty late night.

**Comment by toalina** [comment]
Oh my. Here's some stuff I did:
- Consolidated the NAV stylings (<ul>, <li>, <a>)
- Grouped some of the flex-containers. Ones I couldn't get to were flexbox applied in the media query (DESKTOP). I attempted to move the flexbox rules out of the places where they were in the DESKTOP rules, but because of other specific settings on mobile (not built for flexbox), it conflicted, so I left those alone. 
- Grouped some of the <p> settings

I think that's all. Shaved 20 lines off... Hey not bad!
